NStack/strings/ustring.cs

```diff
@@ -129,9 +128,8 @@ public IntPtrUString (IntPtr block, int size, bool copy, Action<IntPtr> releaseF
 					throw new ArgumentException ("Invalid size passed", nameof (size));
 				this.size = size;
 
-				this.copy = copy;
 				if (copy) {
-					this.release = null;
+					this.release = Marshal.FreeHGlobal;
```

Comment:
None = null, so we do a `?.Invoke()` null check then call. Having an empty delegate would invoke.